test-resources-hibernate-reactive/test-resources-hibernate-reactive-mssql/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'io.micronaut.build.internal.hibernate-reactive-module'
+}
+
+description = """
+Provides support for launching a MSSQL test container for Hibernate Reactive.
+"""
+
+dependencies {
+    implementation(libs.testcontainers.mssql)
+    implementation(project(":test-resources-jdbc-mssql"))
+
+    testRuntimeOnly(libs.mssql)
```

Comment:
No JDBC driver is needed for Hibernate Reactive

Reply:
It is required for the test which verifies that it can run with datasources

Reply:
I would personally have a separate project to test that no dependency is needed.